draft-ietf-regext-rdap-geofeed.md

```diff
@@ -32,107 +32,235 @@ email = "tomh@apnic.net"
+            "method": "removal"
+        }
+    ],
+    ...
 }
 ```
 
 # Privacy Considerations
 
-When including a geofeed file URL in an IP Network object, an RDAP server
-operator SHOULD follow the guidance from [@?I-D.ymbk-opsawg-9092-update, section 7]
-to not accidentally expose the location of an individual. 
+When including a geofeed file URL in an IP Network object, an RDAP server operator SHOULD follow the guidance from
+[@?I-D.ymbk-opsawg-9092-update, section 7] to not accidentally expose the location of an individual. 
 
 # Security Considerations
 
-[@?I-D.ymbk-opsawg-9092-update] requires an HTTPS URL for a geofeed file, and
-optionally RPKI-signing the data within. Besides that, this document does not
-introduce any new security considerations past those already discussed in the
+[@?I-D.ymbk-opsawg-9092-update] requires an HTTPS URL for a geofeed file, and optionally RPKI-signing the data within.
+Besides that, this document does not introduce any new security considerations past those already discussed in the
 RDAP protocol specifications.
 
 # IANA Considerations
 
-IANA is requested to register the following value in the RDAP Extensions
-Registry:
+## RDAP Extensions Registry
+
+IANA is requested to register the following value in the RDAP Extensions Registry:
+
+* Extension identifier: geofeed1
+* Registry operator: Any
+* Published specification: This document.
+* Contact: IETF <iesg@ietf.org>
+* Intended usage: This extension describes version 1 of a method to access the IP geolocation feed data through RDAP.
+
+## Link Relations Registry
+
+IANA is requested to register the following value in the Link Relations Registry:
+
+* Relation Name: geo
+* Description: Indicates that the link context has a resource with geographic information at the link target.
+* Reference: This document.
+
+## Media Types Registry
+
+* Type name: application
+* Subtype name: geofeed+csv
+* Required parameters: N/A
+* Optional parameters: N/A
+* Encoding considerations: See [@?I-D.ymbk-opsawg-9092-update, section 2].
+* Security considerations: See the "Security Considerations" section of this document.
+* Interoperability considerations: There are no known interoperability problems regarding this media format.
+* Published specification: This document.
+* Applications that use this media type: Implementations of the Registration Data Access Protocol (RDAP) Extension for
+Geofeed Data.
+* Additional information: This media type is a product of the IETF REGEXT Working Group. The REGEXT charter, information
+on the REGEXT mailing list, and other documents produced by the REGEXT Working Group can be found at https://datatracker.ietf.org/wg/regext/.
+* Person & email address to contact for further information: IESG <iesg&ietf.org>
+* Intended usage: COMMON
+* Restrictions on usage: None
+* Authors: Tom Harrison, Jasdip Singh
+* Change controller: IETF
+* Provisional Registration: No
+
+## Structured Syntax Suffixes Registry
```

Comment:
Does this look ok?

Reply:
This looks fine to me, in the sense that it aligns with the existing registry entries.  I'm not completely clear on what "cases defined in +csv" means, since I would have thought the process here was to use the logic for the more-specific media type first, before falling back to standard CSV processing, but this is not something I know very much about.

Reply:
Same here. I emulated the entry for "+json". My guess is "cases defined in +csv" means the various fragmentation scenarios for a resource (places within a representation a fragmentation identifier could point to).